lib/index.js

```diff
@@ -46,6 +53,9 @@ export function activate() {
   subscriptions.add(atom.config.observe('linter-stylelint.disableWhenNoConfig', value => {
     disableWhenNoConfig = value;
   }));
+  subscriptions.add(atom.config.observe('linter-stylelint.enableHtmlStyleLinting', value => {
+    enableHtmlStyleLinting = value;
```

Comment:
You need to handle the option being _disabled_ as well, for an example of how `linter-eslint` does this see [here](https://github.com/AtomLinter/linter-eslint/blob/v7.1.3/lib/main.js#L78-L87).